### ISSUE TITLE
wgengine/monitor: IPv4-ism in Linux link monitor

### ISSUE THREAD

**Issue by bradfitz** [tailscale/tailscale#1385]
`monitor_linux.go` has:

```go
                if msg.Header.Type == unix.RTM_NEWROUTE && rmsg.Table == tsTable && rmsg.DstLength == 32 {
                        // Don't log. Spammy and normal to see a bunch of these on start-up,                                                                                                         
                        // which we make ourselves.                                                                                                                                                  
                } else {
```

We should be consistent with IPv6 for 1.6.

/cc @danderson

**Comment by bradfitz** [comment]
sure enough, after updating past df5adb2e2333b9de4e8e46b87d710ea4edbee49c, my local machine spams at startup:

```
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:626c:9632/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6255:9580/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6250:2742/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6266:6435/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:626c:681f/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6249:a456/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:627a:8617/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6278:4a6e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6269:cf0c/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6269:b308/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6272:e024/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624e:ee43/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624c:f308/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:626d:f609/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6251:922e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6266:d83d/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624b:e865/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6263:9c3b/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6241:600b/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:626a:534/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6271:9f78/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6255:d477/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6266:434/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6259:515f/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:625c:2711/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6271:1574/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6260:5635/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6244:9d32/128, gw=, outif=342, table=52
control: successful lite map update in 175ms
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:626a:7d2a/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6273:493c/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6274:fa49/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:625b:285e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6257:fd62/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:626a:8f6a/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6251:c348/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6241:ce1b/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6258:e877/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6269:280e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6263:171c/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6268:e276/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6256:7165/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624e:c129/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6243:1826/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6268:8952/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6274:113e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624d:3e4d/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:625f:4d7c/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6251:1211/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6244:821b/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624b:960d/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:625d:4972/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6243:bf4a/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6278:c463/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:627d:986e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:625f:9261/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:625b:394b/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6269:eb2c/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6246:9103/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:624e:bd2e/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:6243:d460/128, gw=, outif=342, table=52
monitor: RTM_NEWROUTE: src=, dst=fd7a:115c:a1e0:ab12:4843:cd96:627e:883d/128, gw=, outif=342, table=52
```

**Comment by danderson** [comment]
Wheee. Oops. Fixing.

**Comment by danderson** [comment]
Bah, forgot to mark change. Fixed in 6e42430ad8447eb54b793c9ce441ef9dd53f8283.
